docs/xxgloballockreservation.md

```diff
@@ -0,0 +1,101 @@
+
+## Syntax
+
+        -XX:[+|-]GlobalLockReservation
+        -XX:+GlobalLockReservation:<parameter>
+
+| Setting                    | Effect | Default                                                                        |
+|----------------------------|--------|:------------------------------------------------------------------------------:|
+|`-XX:+GlobalLockReservation`| Enable |                                                                                |
+|`-XX:-GlobalLockReservation`| Disable| <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span> |
+
+This optimization is targeted towards applications with lots of uncontended locked objects that are being locked just to be safe. When enabled, heuristics are used to try and determine when an object will be exclusively locked by a single thread so that faster, more specialized code can be used for locking the object. Negative performance side effects may occur if the heuristics used get things wrong but functional correctness should still be maintained.
+
+The `-XX:-GlobalLockReservation` option turns off a previously enabled `-XX:+GlobalLockReservation` option.
+
+## Parameters
+
+| Parameter                                                     | Effect                                                                                          |
+|---------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| [`reservedTransitionThreshold`](#reservedtransitionthreshold) | Advanced tuning parameter. Changes amount of time spent analyzing an object.                    |
+| [`reservedAbsoluteThreshold`  ](#reservedabsolutethreshold  ) | Advanced tuning parameter. Changes amount of time spent analyzing a class for compatibility.    |
+| [`minimumReservedRatio`       ](#minimumreservedratio       ) | Advanced tuning parameter. Changes aggression level for marking a class as highly compatible.   |
+| [`cancelAbsoluteThreshold`    ](#cancelabsolutethreshold    ) | Advanced tuning parameter. Changes amount of time spent analyzing a class for incompatibility.  |
+| [`minimumLearningRatio`       ](#minimumlearningratio       ) | Advanced tuning parameter. Changes aggression level for marking a class as highly incompatible. |
+
+
+### reservedTransitionThreshold
+        -XX:+GlobalLockReservation:reservedTransitionThreshold=<value>
+
+: | Setting       | Value          | Default               |
+  |---------------|----------------|-----------------------|
+  | `<value>`     | number         | 1                     |
+
+: Number of times an object is locked by the same thread before it is considered reserved minus 2. So, with the default value of 1 and object can be reserved the third time it is locked. Value can be 0-3 inclusive. Values of 4 or higher are treated as infinity.
```

Comment:
For clarity, can we say 

"Number of times an object is locked by the same thread before it is considered reserved minus a value of 2. So, with a default value of 1, an object can be reserved the third time it is locked. `<value>` can be 0-3 inclusive. Values of 4 or higher are treated as infinity."

Reply:
That sounds better so I'll change it.